site/about/overview-llm-features.qmd

```diff
@@ -0,0 +1,113 @@
+---
+title: "Large language model features"
+date: last-modified
+filters:
+  - tachyons
+lightbox: auto
+listing:
+  - id: llm-whats-next
+    type: grid
+    max-description-length: 500
+    sort: false
+    grid-columns: 2
+    table-hover: true
+    image-align: left
+    contents:
+    - path: https://validmind.com/about/legal
+      title: Read our legal ease
+      description: We believe in transparency across all our practices — not just because it’s essential to earn your trust, but because it’s the right thing to do.<br><br>Understanding our policies shouldn’t feel like deciphering code, so we’ve made our legal texts as clear and accessible as possible. These documents cover the details of our practices, including data handling, privacy, responsible AI use, and regulatory compliance.
+      # author: validmind.com
+    - path: https://validmind.com/contact/
+      title: Request a demo 
+      image: /assets/img/hero-platform.png
+      description: Discover how ValidMind’s LLM-powered platform, purpose-built for model risk management teams, enables fast, confident testing, documentation, validation, and governance of Generative AI, AI, and statistical models.
+      # author: validmind.com
+---
+
+{{< var vm.product >}} offers several specialized features that use large language models (LLMs) to streamline model risk management and ensure regulatory compliance. Here's how we approach these features and what you need to know. 
+
+
+::: {.attn}
+## {{< fa list-check >}} Our philosophy
+
+<!-- Don't add a caption, it will push the footnote outside the section -->
+::: {.column-margin}
+![](overvierw-llm-features-check-document.png){fig-alt="A screenshot showing the LLM feature for checking documents that can be accessed with the Check Document button" class="screenshot"}
+:::
+
+At ValidMind, our approach to AI features, particularly in testing and risk management, reflects a philosophy of _drinking our own champagne_ or using our own products. This means that, while we create risk management tools for our users, we also rigorously apply these tools and practices within our own product development.
```

Comment:
```suggestion
At {{< var vm.product >}}, our approach to AI features, particularly in testing and risk management, reflects a philosophy of _drinking our own champagne_ or using our own products. This means that, while we create risk management tools for our users, we also rigorously apply these tools and practices within our own product development.
```

Reply:
Couldn't commit this directly for some reason but fixed in the source.